style.css

```diff
@@ -0,0 +1,645 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato&display=swap");
+
+@font-face {
+  font-family: 'cocogoose';
+  src: url(fonts/cocogoose-pro.regular.ttf);
+}
+
+* {
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 16px;
+}
+
+header {
+  position: relative;
+  top: 2%;
+  left: 2%;
+}
+
+header i.menu-btn {
+  font-size: 3rem;
+  cursor: pointer;
+}
+
+nav {
+  position: fixed !important;
+  width: 100vw;
+  top: 0;
+  left: -100000px;
+  -webkit-transition: all 3s 0ms ease-in-out;
+  transition: all 3s 0ms ease-in-out;
+  height: 100vh;
+}
+
+nav i.close-btn {
+  font-size: 2.2rem;
+  cursor: pointer;
+  position: absolute;
+  top: 0;
+  left: 0;
+  padding: 0.2em 0.3em;
+}
+
+nav .nav-item {
+  font-family: "Lato", sans-serif;
+  font-size: 1.3rem;
+  font-weight: bold;
+}
+
+nav .nav-item .nav-link:hover {
+  color: #ec5242 !important;
+}
+
+nav .nav-item .nav-link:active {
+  color: #ec5242 !important;
+}
+
+nav .nav-item .btn-menu {
+  border: 2px solid #ec5242;
+  color: #ec5242 !important;
+  padding: 0.9em;
+  font-size: 1rem;
+  font-family: "Lato", sans-serif;
```

Comment:
- [ ] [OPTIONAL]  I noticed you are using the font Lato in other areas in your code.
It will be a good practice to follow the[ CSS DRY METHOD](https://vanseodesign.com/css/dry-principles/). It help a lot to reduce code duplication.